atomics/T1033/T1033.yaml

```diff
@@ -111,7 +111,6 @@ atomic_tests:
   description: "Identify the system owner or current user using native Windows command prompt utilities."
   supported_platforms:
     - "windows"
-  auto_generated_guid: 35b88076-7edb-4eb5-bdc5-11ede7f45c6a
```

Comment:
This atomic has 2 keys `auto_generated_guid` which is causing validation checks to fail